config/libc/glibc.in

```diff
@@ -68,7 +68,7 @@ config GLIBC_DEP_MAKE_4_0
 # where the build process hangs indefinitely
 config GLIBC_DEP_MAKE_4_3
     def_bool y
-    depends on GLIBC_older_than_2_32 && CONFIGURE_has_make_4_4_or_newer
+    depends on GLIBC_older_than_2_32 && (CONFIGURE_has_make_4_4_or_newer || GLIBC_DEP_MAKE_4_0)
```

Comment:
I think the condition should be `|| !CONFIGURE_has_make_4_0_or_newer`

Reply:
With that change glibc 2.27 or earlier would also start to require make >= 4, which is a bigger change in behaviour.

Reply:
Ah OK. Make sense